src/getSettings.js

```diff
@@ -0,0 +1,10 @@
+
```

Comment:
overkill to have one file just for that, can you put it in `spacebro-client.js` ?